### ISSUE TITLE
How do I access a SQLite file from the docker installation?

### ISSUE THREAD

**Issue by asitemade4u** [dbgate/dbgate#215]
**Details:**
I am trying to access a SQLite file `db.sqlite3` located on a disk attached to the host where DBGate is running as a docker stack.
So:

1. I added a `volumes:` instruction in the `docker-compose.yml` file
2. I bound the volume where the SQLite file is located to `/mnt/sqt/` directory within the container
3. and I created a new `CONNECTION:` in the  `docker-compose.yml` file as:

```yaml
      LABEL_con4: sqt
      FILE_con4: /mnt/sqt/db.sqlite3
      ENGINE_con4: sqlite@dbgate-plugin-sqlite
```
4. I checked in the docker console and the file is indeed in the `/mnt/sqt/` folder of the stack
5. BUT I am unable to access it from DBGate's interface: it is postfixed with a round icon

**Version Information [might be relevant to your issue]**
Operating System: docker on Ubuntu Server LTS 20.04
App Version [help -> about]: latest docker image

**Comment by janproch** [comment]
Your configuration is correct, there was bug in interpreting environment variables for single-file database (SQLite).

It is fixed in newest BETA, release will be next week

**Comment by janproch** [comment]
Released in version 4.6.3

**Comment by asitemade4u** [comment]
I am sorrfy to reopen the topic but even after updating to the latest verfsion I stikll have difficulties accessing a SQLite file.

1. same config as above
2. I changed the permissions of the mounted directory where the database file is located to `root`
3. I can now see the database in DBGate's sinterface allright: it has a specific yellow icon
4. BUT the structure is not displayed: a `Loading database structure` message is displayed indefinitely.

Questions:

- do I have to stop the database that is using the database file? In other terms, is there a lock on SQLite files?
- are the file permissions I am using correct?
